src/extension/importer/rest/src/test/java/org/geoserver/importer/rest/ImporterIntegrationTest.java

```diff
@@ -1174,6 +1174,7 @@ public void testCharsetEncodingOption() throws Exception {
         assertEquals(ImportContext.State.COMPLETE, context.getState());
         assertTrue(context.getState() == ImportContext.State.COMPLETE);
 
-        assertTrue(new File(context.getUploadDirectory().getFile(), ".locking").exists());
+        assertTrue(new File(context.getUploadDirectory().getFile(), "bad_char.shp").exists());
```

Comment:
@afabiani did you work on the locking files before? not sure if removing the test about existance of the locking file is correct or not...